volume/drivers/extpoint_test.go

```diff
@@ -21,3 +22,81 @@ func TestGetDriver(t *testing.T) {
 		t.Fatalf("Expected fake driver, got %s\n", d.Name())
 	}
 }
+
+// testParseMountSpec is a structure used by TestParseMountSpecSplit for
```

Comment:
You were right. Not sure why I thought it had to go here. Moved. Will be in the next push.